cypress/integration/login-test.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable no-undef */
+describe("First Test", function () {
+  it("can log in and out", () => {
+    cy.visit("https://staging.sparkle.space/in/devaliashacksville");
+    cy.clearCookies();
+
+    cy.get(".login-button")
+      .click();
+
+    cy.get(".form > .input-group:nth-child(1) > input")
+      .type('malaika.aiyar@gmail.com');
+
+    cy.get(".form > .input-group:nth-child(2) > input")
+      .type('QWE#qwe3');
```

Comment:
@malaikaaiyar If you haven't already, since this password is exposed now, you should change it.